tests/tools/test_rag_tools.py

```
@@ -1,47 +1,32 @@
-# We'll patch 'epic_news.tools.rag_tools.DEFAULT_RAG_CONFIG' to use this copy.
-# This structure should mirror epic_news.rag_config.DEFAULT_RAG_CONFIG
-TEST_DEFAULT_RAG_CONFIG = {
-    "llm": {
-        "provider": "openai",  # Using a common provider for tests
+# Mirror the production DEFAULT_RAG_CONFIG shape (crewai-tools 1.x).
+TEST_DEFAULT_RAG_CONFIG: dict[str, Any] = {
+    "vectordb": {
+        "provider": "chromadb",
         "config": {
-            "model": "gpt-3.5-turbo",  # Example model
+            "persist_directory": "/tmp/test_chroma_db",
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/test_chroma_db"

(S108)